ci/tlk.ps1

```diff
@@ -823,6 +823,10 @@ class TlkRecipe
             $DpkgBuildPackageArgs += @('-a', $this.Target.DebianArchitecture())
         }
 
+        # Strip first so the included binary is identical for deb and rpm
+        & 'strip' $Executable | Out-Host
+
+        $env:DEB_BUILD_OPTIONS = "nostrip"
```

Comment:
style: It’s PowerShell so technically valid… but usually `$Env` is spelled with a leading uppercase.